### PR TITLE
frame: cqlvalue, added AsStringSlice()

### PR DESCRIPTION
**Pull request by Kulezi**
@Michal-Leszczynski mentioned the need for parsing lists of token strings in cluster, those are actually of `set<text>` type.
This PR adds CqlValue.AsTextSet() allowing parsing token sets.

Also I think that validating correctness of CqlValue underlying bytes (e.g. checking if strings are utf8) should be done only when Debug flag is set, as we shouldn't get invalid data from queries. Correct me if I'm wrong.

### CODE REVIEW

**Comment by mmatczuk** [comment]
Naming, can we use String instead of Text and Slice instead of Set i.e. []string is StringSlice.
This is a general comment applies to some existing code too.

**Comment by mmatczuk** [comment]
Also, technically CqlFromTextSet creates a List?
A more flexible naming convention is needed.

File: frame/cqlvalue.go [hunk 1]
```diff
@@ -198,6 +199,31 @@ func (c CqlValue) AsFloat64() (float64, error) {
 		uint64(c.Value[7])), nil
 }
 
+func (c CqlValue) AsTextSet() ([]string, error) {
+	if c.Type.ID != SetID || c.Type.List.Element.ID != VarcharID {
```
Comment (Michal-Leszczynski) [inline frame/cqlvalue.go:203]: I get errors when using this function. There should be:
`c.Type.Set.Element.ID`
instead of:
`c.Type.List.Element.ID`

**Review by Michal-Leszczynski — CHANGES_REQUESTED** [review]

**Comment by Kulezi** [comment]
Renamed to AsStringSlice(), this feels more natural, it allows parsing from lists and sets of varchar, text and ascii types.

**Review by Choraden — APPROVED** [review]

**Review by Michal-Leszczynski — APPROVED** [review]

**Comment by Kulezi** [comment]
Merged with #179

**Pull request closed without merging**